cockatrice/src/dlg_settings.cpp

```diff
@@ -246,6 +249,7 @@ void GeneralSettingsPage::retranslateUi()
     highQualityURLLabel.setText(tr("Custom Card Download URL:"));
     highQualityURLLinkLabel.setText(QString("<a href='%1'>%2</a>").arg(LINKING_FAQ_URL).arg(tr("Linking FAQ")));
     clearDownloadedPicsButton.setText(tr("Reset/Clear Downloaded Pictures"));
+    updateNotificationCheckBox.setText(tr("Notify when client updates are available"));
```

Comment:
This wording was put in place with the expectation that there would be some type of client update notification system worked in the client other than the feature set communication.  At least that's what @Daenyth was eluding to in the previous update check PR that I put in.  The wording can be changed but I was trying to keep it simple for the user and re-usable at a later date as the only functionality it permits/blocks is the notification message box of a newer version in general available.